src/Item/hashes.ts

```diff
@@ -20,26 +24,26 @@ const IMAGE_PATH = 'image.png'
 function buildStandardWearableEntityMetadata(
   item: ItemAttributes,
   collection: CollectionAttributes
-): StandardWearableEntityMetadata {
+): StandardWearable & { emoteDataV0?: { loop: boolean } } {
   if (!isStandardItemPublished(item, collection)) {
     throw new Error(
       "The item's collection must be published to build its metadata"
     )
   }
 
-  const entity: StandardWearableEntityMetadata = {
+  const entity: StandardWearable & { emoteDataV0?: { loop: boolean } } = {
```

Comment:
That's a great question. I haven't named it because `emoteDataV0` was just something to allow emotes to be launched, but Emotes will have their own entity in the short term, so this should be removed from the `StandardWearable` entity. Should I type it anyways?

Reply:
Naah, I thought it might be because of that. Let's keep it this way then